src/main/java/com/connexta/ddf/catalog/direct/CatalogMethods.java

```diff
@@ -91,7 +99,7 @@
   {
     Builder<String, DocMethod> builder = ImmutableMap.builder();
     builder.put(
-        "ddf.catalog/create",
+        CREATE_KEY,
```

Comment:
Any particular reason this key was moved to it's own variable, but not the other methods?

Reply:
not really, i can revert it if we don't want it there, but ideally they all are extracted out to constants when the unit tests are written.